couchbase/src/test/java/docs/javadsl/Example.java

```diff
@@ -0,0 +1,15 @@
+package docs.javadsl;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.auth.PasswordAuthenticator;
+
+public class Example {
```

Comment:
should there be an example for couchbase3 too?

Reply:
This is an unnecessary change, I reverted. 
The code for example is usually related to the design doc and has some coupling with couchbase2. I plan to place it in another PR. Regarding how to handle Couchhbase2 and Couchhbase3. I hope we can further discuss in the issue.